compiler/luci-interpreter/src/kernels/Softmax.cpp

```diff
@@ -60,5 +68,18 @@ void Softmax::evalFloat() const
                                  getTensorShape(output()), getTensorData<float>(output()));
 }
 
+template <typename T> void Softmax::evalQuantized() const
+{
+  tflite::SoftmaxParams params{};
+  // params.beta = _params.beta;
+  float intermediate[256];
+  params.table = intermediate;
+  params.zero_point = output()->zero_point();
+  params.scale = output()->scale();
+  tflite::optimized_ops::PopulateSoftmaxLookupTable(&params, input()->scale(), _params.beta);
```

Comment:
Why not create the lookup table in `configure` and reuse it?

Reply:
Oh, I will fix it.

Reply:
Okay I will fix it.